bot.py

```diff
@@ -128,6 +128,13 @@ def process_message_inner(config, message, conv):
         conv.set_debug(False)
         return keybase.send(conv.id, NODEBUG)
 
+    elif msg_type == parse.MSG_SNOOZE:
+        if conv.context != conversation.CTX_REMINDED:
+            return keybase.send(conv.id, "Not sure what to snooze.")
+        conv.get_reminder().snooze_until(data.time)
+        conv.clear_weak_context()
```

Comment:
Maybe we actually want to set CTX_SET because we've just set a reminder now and the user might say "thank you"